chainerrl/wrappers/continuing_time_limit.py

```diff
@@ -38,8 +38,9 @@ def step(self, action):
         observation, reward, done, info = self.env.step(action)
         self._elapsed_steps += 1
 
-        if self._max_episode_steps <= self._elapsed_steps:
-            info['needs_reset'] = True
+        if self._max_episode_steps:
+            if self._max_episode_steps <= self._elapsed_steps:
+                info['needs_reset'] = True
```

Comment:
If `max_episode_steps` is None, it does not make sense to wrap env with `ContinuingTimeLimit`. I think it is better to modify `make_atari` so that it skips `ContinuingTimeLimit` if `max_frames` is None, not modifying `ContinuingTimeLimit`.